framework/src/utils/RandomGenerator.C

```diff
@@ -0,0 +1,45 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#include "RandomGenerator.h"
+#include  <random>
+
+class MersenneTwister
+{
+public:
+  std::mt19937 _mt_eng;
+};
```

Comment:
What we'll want to do is create a wrapper for random number generation. You don't want to use random numbers directly from the generator for one big reason in MOOSE: parallel and threaded reproducibility. I know the use case is different with the tools you are building but with a mesh and spatial variables, we want to make sure that a particular analysis doesn't change as you vary the number of processors. This is primary purpose of `MooseRandom`.

I'd prefer that if you want to use some of the C++11 generators that you add to the existing interface. I'm fine adding new methods to the old interface, or adding options so that people can switch out which generator is active, etc.